anyway/widgets/suburban_widgets/accident_count_by_driver_type_widget.py

```diff
@@ -29,9 +29,11 @@ def generate_items(self) -> None:
 
     @staticmethod
     def count_accidents_by_driver_type(request_params):
+        filters = get_injured_filters(request_params.location_info)
+        filters["involved_type"] = [1,2]

```

Comment:
looks good, perhaps we should add it to our constants (backend_constants.py file) ?